### PR TITLE
chore(agw): Bazelify the load_tests

### LINKED ISSUE #13813: Investigate state of `load_test`s and relevance for bazel migration
- where are the load tests currently used and how are they setup?
- how could a bazel setup look like?

timeboxed: 2

### PR DESCRIPTION
**Pull request by LKreutzer**
Signed-off-by: Lars Kreutzer <lars.kreutzer@tngtech.com>

## Summary

- Bazelify the load tests 
- Create script to run them
- Include newer directoryd load test which was not runnable so far
- Remove Make/fabric commands to run load tests
- Remove load tests from bazel magma.deb creation
- Resolves #13813

## Test Plan

-  Start magma vm and build and run services
- Run `$MAGMA_ROOT/bazel/scripts/run_load_tests.sh` 
## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

File: lte/gateway/python/load_tests/README.md [hunk 1]
```diff
@@ -16,34 +16,25 @@ failure, and so on. The options for configuration and such can be looked
...
-   `vagrant up magma && vagrant ssh magma`
-2. Now in the gateway VM: `cd $MAGMA_ROOT/lte/gateway && make run`
+Next build and start the magma services. See the [integ test documentation](https://docs.magmacore.org/docs/next/lte/s1ap_tests#s1ap-integration-tests) for details.
 
 ### Run tests
 
-From `$MAGMA_ROOT/lte/gateway/python/load_tests` on the *magma* VM, run:
-
-* All tests: `make load_test`
+On the gateway VM, run `$MAGMA_ROOT/bazel/scripts/run_load_tests.sh` from anywhere inside the magma folder.
```
Comment (sebathomas) [inline lte/gateway/python/load_tests/README.md:25]: I always get an error when I try to execute this (no matter if I built magma with make or bazel):
```
INFO: Elapsed time: 12.587s, Critical Path: 1.85s
INFO: 1671 processes: 1629 disk cache hit, 13 internal, 29 linux-sandbox.
INFO: Build completed successfully, 1671 total actions
Finished building load tests.
#############################
Running load tests...
#############################
Running load test: loadtest_mobilityd allocate
sudo: /home/vagrant/magma/bazel-bin/lte/gateway/python/load_tests/loadtest_mobilityd allocate: command not found
```

Maybe there is a problem running Bazel while you're in the virtualenv? I just restarted a build...

File: bazel/scripts/run_load_tests.sh [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    echo "#############################"
+}
+
+run_load_tests() {
+    echo "Running load tests..."
+    for LOAD_TEST in "${LOAD_TEST_LIST[@]}";
+    do  
+        echo "#############################"
+        echo "Running load test: ${LOAD_TEST}"
+        # shellcheck disable=SC2086
```
Comment (LKreutzer) [inline bazel/scripts/run_load_tests.sh:35]: This is to make sure `LOAD_TEST` can be interpreted as two arguments here without increasing the complexity of the script too much. The check is only disabled for the next line.

File: lte/gateway/python/load_tests/README.md (hunk 1 above)
Comment (LKreutzer) [reply lte/gateway/python/load_tests/README.md:25]: Sorry about that, I was just fixing a shellcheck warning which caused this. I fixed it - see comment in the script.

Comment (LKreutzer) [reply lte/gateway/python/load_tests/README.md:25]: The issue was that it was looking for the file `"home/vagrant/magma/bazel-bin/lte/gateway/python/load_tests/loadtest_mobilityd allocate"` which does not exist with the ` allocate` in the same string.

**Comment by LKreutzer** [comment]
FYI the warning
`/home/vagrant/magma/bazel-bin/lte/gateway/python/load_tests/loadtest_pipelined.runfiles/ryu_repo/ryu/lib/packet/cfm.py:271: SyntaxWarning: "is not" with a literal. Did you mean "!="?
  assert interval is not 0`
can be ignored. It is ryu internal and does not seem to cause any issues.

**Comment by LKreutzer** [comment]
Expected output is:
```
vagrant@magma-dev:~/magma$ $MAGMA_ROOT/bazel/scripts/run_load_tests.sh
Building load tests...
INFO: Invocation ID: 4d8c9489-ad76-4a51-a9a4-5bd88784d29a
INFO: Reading 'startup' options from /home/vagrant/magma/.bazelrc: --output_base=/tmp/bazel, --host_jvm_args=-Xmx8g
INFO: Options provided by the client:
  Inherited 'common' options: --isatty=1 --terminal_columns=207
INFO: Reading rc options for 'build' from /home/vagrant/magma/bazel/bazelrcs/cache.bazelrc:
  Inherited 'common' options: --config=disk_cache
INFO: Reading rc options for 'build' from /home/vagrant/magma/.bazelrc:
  'build' options: --announce_rc --color=yes --cxxopt=-std=c++14 --strip=never --per_file_copt=^lte/gateway/c/.*$@-g --define=folly_so=1 --test_env=PATH=/bin:/usr/bin:/usr/local/bin:/usr/sbin --test_env=MAGMA_ROOT --test_env=S1AP_TESTER_ROOT --per_file_copt=^lte/gateway/c/core/.*$@-DPACKAGE_BUGREPORT="TBD" --per_file_copt=^lte/gateway/c/core/.*$@-DPACKAGE_VERSION="0.1" --per_file_copt=^lte/gateway/c/core/oai/tasks/amf/.*$@-DPACKAGE_NAME="AMF" --per_file_copt=^lte/gateway/c/core/oai/tasks/mme_app/.*$@-DPACKAGE_NAME="MME" --per_file_copt=^lte/gateway/c/core/oai/tasks/sgw/.*$@-DPACKAGE_NAME="S/P-GW" --javacopt=-source 8 --javacopt=-target 8 --java_runtime_version=remotejdk_11
INFO: Reading rc options for 'build' from /home/vagrant/magma/bazel/bazelrcs/vm.bazelrc:
  'build' options: --config=vm
INFO: Found applicable config definition common:disk_cache in file /home/vagrant/magma/.bazelrc: --repository_cache=/var/cache/bazel-cache-repo
INFO: Found applicable config definition build:disk_cache in file /home/vagrant/magma/.bazelrc: --disk_cache=/var/cache/bazel-cache
INFO: Found applicable config definition build:vm in file /home/vagrant/magma/.bazelrc: --config=specify_vm_cc
INFO: Found applicable config definition build:specify_vm_cc in file /home/vagrant/magma/.bazelrc: --action_env=CC=/usr/bin/gcc
INFO: Analyzed 7 targets (0 packages loaded, 0 targets configured).
INFO: Found 7 targets...
INFO: Elapsed time: 0.119s, Critical Path: 0.00s
INFO: 1 process: 1 internal.
INFO: Build completed successfully, 1 total action
Finished building load tests.
#############################
Running load tests...
#############################
Running load test: loadtest_mobilityd allocate
#############################
Running load test: loadtest_mobilityd release
#############################
Running load test: loadtest_pipelined activate_flows
/home/vagrant/magma/bazel-bin/lte/gateway/python/load_tests/loadtest_pipelined.runfiles/ryu_repo/ryu/lib/packet/cfm.py:271: SyntaxWarning: "is not" with a literal. Did you mean "!="?
  assert interval is not 0
QOS Enabled
#############################
Running load test: loadtest_pipelined deactivate_flows
/home/vagrant/magma/bazel-bin/lte/gateway/python/load_tests/loadtest_pipelined.runfiles/ryu_repo/ryu/lib/packet/cfm.py:271: SyntaxWarning: "is not" with a literal. Did you mean "!="?
  assert interval is not 0
#############################
Running load test: loadtest_sessiond create
#############################
Running load test: loadtest_sessiond end
#############################
Running load test: loadtest_subscriberdb add
#############################
Running load test: loadtest_subscriberdb list
#############################
Running load test: loadtest_subscriberdb delete
#############################
Running load test: loadtest_subscriberdb get
#############################
Running load test: loadtest_subscriberdb update
#############################
Running load test: loadtest_policydb enable_static_rules
#############################
Running load test: loadtest_policydb disable_static_rules
#############################
Running load test: loadtest_directoryd update_record
#############################
Running load test: loadtest_directoryd delete_record
#############################
Running load test: loadtest_directoryd get_record
#############################
Running load test: loadtest_directoryd get_all_records
#############################
Finished running load tests.
The result JSON files (result_<name_of_grpc_function>.json) can be found in /var/tmp.
```

**Review by sebathomas — APPROVED** [review]
Was able to run the tests on my machine too. (They really generate a lot of data in Redis. :))

**Review by nstng — APPROVED** [review]
works great - tested locally

**Pull request merged**